raiden/utils/cli.py

```diff
@@ -301,5 +310,78 @@ def expand_default(default, params):
         return HypenTemplate(default).substitute(params)
 
 
+def apply_config_file(
+    command_function: Union[click.Command, click.Group],
+    cli_params: Dict[str, Any],
+    ctx,
+    config_file_option_name='config_file',
+):
+    """ Applies all options set in the config file to `cli_params` """
+    paramname_to_param = {param.name: param for param in command_function.params}
+    path_params = {
+        param.name
+        for param in command_function.params
+        if isinstance(param.type, (click.Path, click.File))
+    }
+
+    config_file_path = Path(cli_params[config_file_option_name])
+    config_file_values = dict()
+    try:
+        with config_file_path.open() as config_file:
```

Comment:
what about moving the config read to a separate function and giving to this function the config dictionary?